src/App.js

```diff
@@ -1,35 +1,38 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import { BrowserRouter, Route } from "react-router-dom";
+import "./App.css";
+import Header from "./components/Header";
+import Home from "./components/Home";
+import Project from "./components/Project";
+import WorksList from "./components/WorksList";
+import data from "./data.json";
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
+    let project;
+    data.forEach((proj) => {
+        return (project = proj);
+    });
+    let { id, year, title, img, technologies } = project;
+    return (
+        <BrowserRouter>
+            <div className="App">
+                <header className="App-header">
+                    <Header />
+                </header>
```

Comment:
it is better if you just have <Header/ > and inside your header component you have the html header tag .. the app component  should be as clean as possible